ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/StreamQueryResponseHandler.java

```diff
@@ -47,9 +42,12 @@
 
   @Override
   protected void handleMetadata(final QueryResponseMetadata queryResponseMetadata) {
-    this.queryResult = new StreamedQueryResultImpl(context, queryResponseMetadata.queryId,
-        Collections.unmodifiableList(queryResponseMetadata.columnNames),
-        RowUtil.columnTypesFromStrings(queryResponseMetadata.columnTypes));
+    this.queryResult = new StreamedQueryResultImpl(
+        context,
+        queryResponseMetadata.queryId,
+        queryResponseMetadata.columnNames,
+        RowUtil.columnTypesFromStrings(queryResponseMetadata.columnTypes)
+    );
```

Comment:
fix formatting

Reply:
Because it's redundant. `queryResponseMetadata.columnNames` is already an unmodifiable list.